webhooks/common/utils.go

```diff
@@ -46,6 +47,7 @@ func RetrieveDefaultImagePublishContentLibrary(ctx context.Context, c ctrlclient
 		for i, cl := range clList.Items {
 			clNames[i] = cl.Name
 		}
+		sort.Strings(clNames)
```

Comment:
It's an easy fix but do you think is it necessary to put burden on the controller logic? Maybe it's better to make the test more accurate?